jwst/rscd/rscd_sub.py

```diff
@@ -185,12 +291,28 @@ def do_correction(input_model, rscd_model):
 
 def get_rscd_parameters(input_model, rscd_model):
 
+
     """
     Read in the parameters from the reference file
-    Seperate these parameters based on even and odd rows
-
     Store the parameters in a param dictionary
+
+    Parameters
+    ----------
+    input_model: data model object
```

Comment:
Could substitute:
```
~jwst.datamodels.RampModel
```
for the generic statement "data model object". That creates a live link to the data model specs.

Reply:
Done.  Here and all the other places in this file.